src/data/hassio.ts

```diff
@@ -129,7 +138,10 @@ export interface HassioSnapshotDetail extends HassioSnapshot {
   repositories: string[];
   folders: string[];
 }
-
+export interface HassioAddonSetOptionParams {
+  audio_input?: string | null;
+  audio_output?: string | null;
```

Comment:
I think these are not optional?

Reply:
They will be optional when I plan to reuse that interface for other options.